### DISCUSSION TITLE
Request to create a plugin for this video service [D*S*T*V]

### DISCUSSION THREAD

**Discussion by esuo2** [streamlink/streamlink#5766] [General]
### Description

Please I would humbly request the team of experts to kindly consider creating a plugin for the video service d*s*t*v Africa
here is the link  h**ps://dstv.stream/

Regards.

**Comment by bastimeyer** [comment]
Please read the rules before posting
https://github.com/streamlink/streamlink/blob/master/.github/DISCUSSION_TEMPLATE/general.yml
https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#plugin-requests
